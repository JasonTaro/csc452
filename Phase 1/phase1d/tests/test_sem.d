test_sem.o: tests/test_sem.c ../phase1.h /Users/kyle/include/usloss.h \
  /Users/kyle/include/usyscall.h
