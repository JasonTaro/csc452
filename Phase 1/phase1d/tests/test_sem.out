 
---------Starting Test SEM ----------
child 1 
 ---------Ending Test SEM ----------
Sentinel quitting.
No runnable processes, halting.
